paperqa/agents/search.py

```diff
@@ -258,9 +258,12 @@ async def index_files(self) -> dict[str, str]:
             if await file_index_path.exists():
                 async with await anyio.open_file(file_index_path, "rb") as f:
                     content = await f.read()
-                    self._index_files = pickle.loads(  # noqa: S301
-                        zlib.decompress(content)
-                    )
+                    try:
+                        self._index_files = pickle.loads(  # noqa: S301
+                            zlib.decompress(content)
+                        )
+                    except Exception:
+                        logger.exception(f"Failed to load index file {file_index_path}")
```

Comment:
```suggestion
                        logger.exception(f"Failed to load index file {file_index_path}.")
```

Adding a period can help reveals empty string issues